zerver/openapi/zulip.yaml

```diff
@@ -5259,27 +5259,30 @@ paths:
         - name: narrow
           in: query
           description: |
-            The narrow where you want to fetch the messages from. See how to
+            The narrow filter where you want to fetch the messages from. See how to
             [construct a narrow](/api/construct-narrow).
```

Comment:
I think this is actually clearer as just "narrow". (Similarly the other two `narrow` parameters below.) I think of the whole thing as "a narrow", consisting of a list of filters.

Calling the whole thing "a narrow" is also consistent with the other text here, and with the parameter's name `narrow`.